### PR TITLE
feat: Created Homepage and Explore Page

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
image

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import "./App.css";
+import { BiSearchAlt } from "react-icons/bi";
+import LandingPage from "./Pages/LandingPage";
+import { Routes, Route } from "react-router-dom";
+import ExplorePage from "./Pages/ExplorePage";
```
Comment (harshitadaswani) [inline src/App.js:5]: you can import `LandingPage` and `ExplorePage` components directly together from the `index.js` file which will optimize your code better

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+function App() {
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="explore" element={<ExplorePage />} />
+      </Routes>
+    </>
```
Comment (harshitadaswani) [inline src/App.js:13]: can extract this into the `AppRouter` component and import it here

File: src/Pages/ExplorePage.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "./../components/Navbar";
+import ExplorePageBody from "./../components/Explore Page/ExplorePageBody";
+
+const ExplorePage = () => {
+  return (
+    <>
+      <Navbar />
```
Comment (harshitadaswani) [inline src/Pages/ExplorePage.jsx:8]: If `Navbar` is common in every component then you can import it in `App.js` to reduce redundancy in the code

File: src/Pages/LandingPage.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+import LandingPageBanner from "./../components/LandingPageBanner";
+
+const LandingPage = () => {
+  return (
+    <>
+      <Navbar />
```
Comment (harshitadaswani) [inline src/Pages/LandingPage.jsx:8]: same as above

File: src/Stylesheets/navbar.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
+.navbar {
+
+    height: 7vh;
+    width: 100vw;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    place-items: center;
+    background-color: #2e2a2a;
+    border-bottom: 2px solid black;
```
Comment (harshitadaswani) [inline src/Stylesheets/navbar.css:9]: can use color variables here

**Review by harshitadaswani — APPROVED** [review]
I liked how you have defined sidebar content and then mapped it to optimize your code. However, there are some suggestions from my side to make your code more optimized, do check!

File: src/Stylesheets/navbar.css (hunk 5 above)
Comment (Kothalkarpriya) [reply src/Stylesheets/navbar.css:9]: yes do declare them in root and access

File: src/Stylesheets/navbar.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+}
+
+.btn__login {
+    width: 8rem;
+    background-color: #F3B2F1;
```
Comment (Kothalkarpriya) [inline src/Stylesheets/navbar.css:15]: here also for colors

File: src/Stylesheets/navbar.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.heading__landingPage h2 {
+    font-weight: 300;
+}
+
+.btn__landingPage {
+    height: 8rem;
+    width: 55rem;
+    background-color: #ff006d;
```
Comment (Kothalkarpriya) [inline src/Stylesheets/navbar.css:62]: as said before for colors

**Review by Kothalkarpriya — APPROVED** [review]
Code is clean and understandable, just use colors as variables by declaring them in the root.

**Pull request merged**